### DISCUSSION TITLE
Microsoft account Sync settings cannot be used.

### DISCUSSION THREAD

**Discussion by jerviscui** [community/community#37977] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/6396748/199239984-6c576512-ba67-4489-b5d8-f75644f6b48a.png)
Microsoft account Sync settings cannot be used.

**Comment by ossanna16** [comment]
Hi there @jerviscui and welcome to our community! Thank you for asking a great question 🙂 Can you please elaborate on your question further so we can help?

**Comment by jerviscui** [comment]
In the [vscode.dev](https://vscode.dev/), I want to use a Microsoft account for sync settings. Then I get the error.
